nabweb/locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -132,17 +131,17 @@ msgstr "animations visuelles Qualité de l'air"
 
 #: templates/nabweb/help.html:12
 msgid "Talking to the rabbit (speech recognition)"
-msgstr "Commnent parler au lapin (reconnaissance vocale)"
+msgstr "Comment parler au lapin (reconnaissance vocale)"
 
 #: templates/nabweb/help.html:12
 msgid ""
 "You need to push and hold the button on the rabbit's head long enough to "
-"hear a 'beep', then wait a couple of seconds and speak. When you are done "
+"hear a sound, then wait a couple of seconds and speak. When you are done "
 "just release the button."
 msgstr ""
 "Il vous faut maintenir le bouton sur la tête du lapin appuyé assez longtemps "
-"pour entendre le 'bip', attendre quelques secondes puis parler. Relachez "
-"quand vous avez terminé."
+"pour entendre un son, attendre quelques secondes puis parler. Relachez quand "
```

Comment:
I do not really agree with this change.
Even if 'beep'/'bip' is more colloquial than sound/son, I think it better coneys the idea that user has to wait for a specific (not any) sound.